tests/installation/start_install.pm

```diff
@@ -31,9 +31,10 @@ sub run {
     # Also, virtual machines for testing can be really slow in this step
     my $started_timeout = get_var('LIVECD') ? 1200 : 300;
     if (is_upgrade) {
+        wait_still_screen 2;
         send_key $cmd{update};
         sleep 1;
```

Comment:
Why `sleep 1` here?

Reply:
It is the original code, I think sleep here just try to ensure the key sent can be received.

Reply:
I also wonder about that `sleep`. That test module is running on the worker, so why do we put a sleep in here. We increased the timeout of the following `assert_screen` from 5 to 10 seconds, to what is the difference betweern sleeping and assert_screen for 10 seconds?

Reply:
I removed the sleep since it is not needed, we have already increased the wait time for assert_screen.